src/main/cpp/subsystems/LimeLightUtil.cpp

```diff
@@ -0,0 +1,50 @@
+#include "subsystems/LimeLightUtil.h"
```

Comment:
Nitpick: Per common practice, the case of the file name should match the case of the class:

```suggestion
#include "subsystems/LimelightUtil.h"
```

I suppose it's possible, by the way, that **did** change the case of the file name on disk, but git didn't pick up the change because Windows path names are not case sensitive. If that's the case, we will need to do some minor wizardry to force it to pick up the change.